src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: ");
+    }

```

Comment:
we dont need to implement it here
```suggestion
```

Reply:
But why not if we have every output starting from "Figure: ..."?